packages/cli/message.txt

```diff
@@ -0,0 +1,18 @@
+	
+	███╗   ███╗███████╗███████╗ ██████╗ 
```

Comment:
could you remove all the tabs in this file, it will look better in the terminal

```suggestion
███╗   ███╗███████╗███████╗ ██████╗ 
```

Reply:
done